documentation/meeting-logs/2023-11-20.md

```diff
@@ -0,0 +1,56 @@
+# Standup Meeting 10
+
+2023-11-20
+Sprint 4
+
+Meeting duration: 1 hour
+
+## Attendees
+
+All group members
+
+## Updates
+
+Dan:
+
+- Hooked up SonarCloud for continuous integration
+  - [https://github.com/BarbzCodez/Spendr/issues/16](https://github.com/BarbzCodez/Spendr/issues/16)
+- This week Dan is going to work on:
+  - Create docs + content for the presentation
+
+Victoria:
+
+- Initial Front End Technical Cleanup
+  - [Front End Cleanup part 2 #143](https://github.com/BarbzCodez/Spendr/issues/143)
+- This week Victoria is going to work on:
+  - Cleanup front end UI so it looks nicer for the presentation
+  - Work on the slides for the presentation
+
+Barbara:
+
+- Initial Front end Cleanup
+  - [Front End Cleanup part 1 #142](https://github.com/BarbzCodez/Spendr/issues/142)
+- Helped Dan with security analysis because I was the repo owner
+- Clean code smells from sonar cube
+  - [Clean Code Smells from SonarCloud #152](https://github.com/BarbzCodez/Spendr/issues/152)
+  - Waiting for the SonarCloud to be merged so that it can run on my branch
+- This week Barbara is going to work on:
+  - Work on the slides for the presentation
+  - Want to implement reset passwords before the end of the sprint
+
+Mark:
+
+- This week Mark is going to work on:
+  - Load Testing Hookup
+  - [Load testing #17](https://github.com/BarbzCodez/Spendr/issues/17)
+
+Ethan:
+
+- Finished writing acceptance tests
+
+## Notes
+
+- Do the presentation with Christmas themed
+  - Victoria starting on the slides
+- Dan is going to start on the content for the presentation in the next couples days
+- We are all wearing turtle necks for the presentation
```

Comment:
love the commitment on this one

Reply:
we are doing it!! Im buying candy cane too